.travis.yml

```diff
@@ -1,82 +1,49 @@
 language: python
-
-# Caching all the depencencies so that they don't have to be redownloaded and
-# compiled everytime
 cache:
   directories:
-    - $TRAVIS_BUILD_DIR/build/GSL/src
-    - $TRAVIS_BUILD_DIR/build/FFTW/src
-    - $TRAVIS_BUILD_DIR/build/SWIG/src
-    - $TRAVIS_BUILD_DIR/build/CLASS/src
-    - $TRAVIS_BUILD_DIR/build/ANGPOW/src
-    - $HOME/miniconda
-
-# Definition of python build matrix for osx comes from:
-# https://docs.travis-ci.com/user/multi-os/#Python-example-(unsupported-languages)
+  - "$TRAVIS_BUILD_DIR/build/GSL/src"
+  - "$TRAVIS_BUILD_DIR/build/FFTW/src"
+  - "$TRAVIS_BUILD_DIR/build/SWIG/src"
+  - "$TRAVIS_BUILD_DIR/build/CLASS/src"
+  - "$TRAVIS_BUILD_DIR/build/ANGPOW/src"
+  - "$HOME/miniconda"
 matrix:
-    include:
-        - os: linux
-          python: 2.7
-          sudo: required
-          env: TOXENV=py27
-        - os: linux
-          sudo: required
-          python: 3.6
-          env: TOXENV=py36
-        - os: osx
-          language: generic
-          env: TOXENV=py27
-        - os: osx
-          language: generic
-          env: TOXENV=py36
-
-#addons:
```

Comment:
I think that this part was to deploy the paper stuff in travis but, I guess it's not necessary.

Reply:
got erased automatically by the travis script, and was commented out anyway so I figured it wasn't really necessary